src/Jmikola/AutoLogin/Http/Firewall/AutoLoginListener.php

```diff
@@ -8,16 +8,14 @@
 use Psr\Log\LoggerInterface;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpKernel\Event\GetResponseEvent;
-use Symfony\Component\Security\Core\SecurityContextInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
 use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
 use Symfony\Component\Security\Core\Authentication\AuthenticationManagerInterface;
 use Symfony\Component\Security\Core\Exception\AuthenticationException;
 use Symfony\Component\Security\Http\SecurityEvents;
 use Symfony\Component\Security\Http\Event\InteractiveLoginEvent;
-use Symfony\Component\Security\Http\Firewall\ListenerInterface;
 
-class AutoLoginListener implements ListenerInterface
+class AutoLoginListener
```

Comment:
I've noted that this is the result of https://github.com/symfony/symfony/pull/28920.

Reply:
Indeed, I have converted this listener to callable to comply with this change.